xdem/spstats.py

```diff
@@ -885,22 +886,29 @@ def patches_method(dh : np.ndarray, mask: np.ndarray[bool], gsd : float, area_si
     return df
 
 
-def plot_vgm(df: pd.DataFrame, fit_fun: Callable = None):
+def plot_vgm(df: pd.DataFrame, list_fit_fun: Optional[list[Callable]] = None, list_fit_fun_label: Optional[list[str]] = None):
```

Comment:
As mentioned in one of your 805920 other PRs ;), `Callable` needs an argument and return type annotation.

Reply:
Double-checking:
If the function is vectorized, should I use `Callable[[np.ndarray],np.ndarray]` to describe this, or rather `Callable[[float],float]` to describe the type of input needed (and forgetting about the fact that it's vectorized)?